docs/docker.md

```diff
@@ -0,0 +1,8 @@
+# Deploy Rems in docker
```

Comment:
could mention this file in the Dockerfile

Reply:
Added a mention to Dockerfile